### ISSUE TITLE
Persistence mechanism is broken

### ISSUE THREAD

**Issue by GoogleCodeExporter** [irishuang1211/macosx-nosleep-extension#10]
```
What steps will reproduce the problem?
1. Install NoSleep
2. Look at the current status - remember it
3. Reboot computer
4. Check the status and confirm that it same as before reboot
5. Change status and perform reboot again
6. Look the status - it is the same as in point 2. <<< This is erroneous 
behavior

What is the expected output? What do you see instead?
Activation status should be the same as before reboot.

```

Original issue reported on code.google.com by `integral...@gmail.com` on 8 Oct 2011 at 9:13

**Comment by GoogleCodeExporter** [comment]
```
Issue 3 has been merged into this issue.
```

Original comment by `integral...@gmail.com` on 8 Oct 2011 at 9:17

**Comment by GoogleCodeExporter** [comment]
```
Issue 8 has been merged into this issue.
```

Original comment by `integral...@gmail.com` on 8 Oct 2011 at 9:17

**Comment by GoogleCodeExporter** [comment]
```
Issue 7 has been merged into this issue.
```

Original comment by `integral...@gmail.com` on 8 Oct 2011 at 9:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `integral...@gmail.com` on 8 Oct 2011 at 9:23
- Changed title: **Persistence mechanism is broken**

**Comment by GoogleCodeExporter** [comment]
```
Yes, this issue is important to me. It would be very nice if fixed :)
```

Original comment by `bsprec...@gmail.com` on 15 Dec 2011 at 2:43

**Comment by GoogleCodeExporter** [comment]
```
Issue 16 has been merged into this issue.
```

Original comment by `integral...@gmail.com` on 15 Feb 2012 at 7:57

**Comment by GoogleCodeExporter** [comment]
```
Fix will be introduced in version 1.3
```

Original comment by `integral...@gmail.com` on 17 Feb 2012 at 5:35
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Issue 18 has been merged into this issue.
```

Original comment by `integral...@gmail.com` on 18 Feb 2012 at 7:17
